benchmarks/benchmark-ab.py

```diff
@@ -130,14 +132,21 @@ def check_torchserve_health():
 def warm_up():
     register_model()
 
+    if execution_params['url'].endswith('.war'):
+        execution_params['inference_model_url'] = 'wfpredict/benchmark'
+
     click.secho("\n\nExecuting warm-up ...", fg='green')
+
     ab_cmd = f"ab -c {execution_params['concurrency']}  -n {execution_params['requests']/10} -k -p {TMP_DIR}/benchmark/input -T " \
              f"{execution_params['content_type']} {execution_params['inference_url']}/{execution_params['inference_model_url']} > {result_file}"
     
     execute(ab_cmd, wait=True)
 
 
 def run_benchmark():
+    if execution_params['url'].endswith('.war'):
```

Comment:
Extract this into a method `is_workflow_run` or something similar, so that in the future if we decide to change the extension for workflow artifacts we won't have to change all if-else statements